qa/standalone/osd/osd-rep-recov-eio.sh

```diff
@@ -120,6 +120,7 @@ function rados_get_data() {
     inject_$inject rep data $poolname $objname $dir 0 || return 1
     rados_get $dir $poolname $objname || return 1
 
+    sleep 1
```

Comment:
Is there any way to poll whether the repair is finished?

Reply:
Switched to wait_for_clean().